agent/dockerclient/dockerapi/docker_client_test.go

```diff
@@ -65,6 +66,15 @@ const xContainerShortTimeout = 1 * time.Millisecond
 // upon the expiration of the timeout duration.
 const xImageShortTimeout = 1 * time.Millisecond
 
+const (
+	// retry settings for pulling images mock backoff
+	xMaximumPullRetries        = 5
+	xMinimumPullRetryDelay     = 25 * time.Millisecond
+	xMaximumPullRetryDelay     = 100 * time.Microsecond
+	xPullRetryDelayMultiplier  = 2
+	xPullRetryJitterMultiplier = 0.2
+)
+
```

Comment:
Should these value match with the ones in `docker_client`? 

If no, how are these decided?

If yes, should we keep a constant to be reused at both the places.

Reply:
I was deciding wether to increase unit test timeout vs use a customized unit test backoff mechanism. I decided to use different backoff values for unit test is because that if we are using same values from docker client, it will make the unit test of this file to run ~60s. I noticed that our test already takes quite long to run, using customized value for unit might be a better option.